core/src/main/res/layout/item_help.xml

```diff
@@ -10,7 +10,7 @@
     android:id="@+id/item_help_title"
     android:layout_width="0dp"
     android:layout_height="wrap_content"
-    android:textAppearance="?android:attr/textAppearanceMedium"
+    android:textAppearance="@style/TextAppearance.KiwixTheme.Headline6"
```

Comment:
`?textAppearanceheadline6` will suffice

We set this attribute in `themes` to be used like this

Reply:
@macgills fixed it. I'll remember this.